### ISSUE TITLE
compilation of pdf fails on the preview service

### ISSUE THREAD

**Issue by lunebellec** [neurolibre/neurolibre#36]
I tried with two repos, including this [one](https://github.com/Notebook-Factory/myelin-meta-analysis) and received the same error message:

``` 
Your preprint build failed to compile with the following errors:

Looks like we failed to compile the PDF with the following error: pandoc: /app/vendor/bundle/ruby/2.6.0/gems/roboneuro-0.1.14/resources/NeuroLibre/latex.template: openBinaryFile: does not exist (No such file or directory) 
``` 
Seems like a problem with the service itself.

**Comment by agahkarakuzu** [comment]
@pbellec thank you for catching this!
 
https://github.com/neurolibre/roboneuro/pull/28/commits/1ff2f1ee1ad18a65673a12ede3c9d260334fc752 resolves the problem. I tested it with [this](https://github.com/Notebook-Factory/myelin-meta-analysis) repo, and it works. 

BTW, I'll add the log functionality I implemented for GitHub calls to the web page calls. Before that, notebook build outputs won't be much informative.
